### ISSUE TITLE
vmalert: extraParams (extra_label) changed behaviour if more than one is defined

### ISSUE THREAD

**Issue by robasp** [VictoriaMetrics/VictoriaMetrics#4908]
### Describe the bug

Hi,
Today, I wanted to upgrade vmalert from 1.90.0 to 1.91.3.
After Upgrading from 1.91.0 to 1.91.2 i noticed, that the extraParams in my alertgroups started to behave differently:
This is my alertgroup:

```
---
groups:
- name: node_ec2
  interval: 10s
  params:
    extra_label:
    - "abc=xyz"
    - "foo=bar"

  rules:

  # CPU
  - alert: ec2CPUUsage95%
    expr: ((avg without (mode, cpu) (1 - rate(node_cpu_seconds_total{mode="idle"}[1m])))*100) > 95
    labels:
      severity: 'info'
...
```
Before 1.91.2, this would result in alerts only firing if the timeseries contained the labels abc=xyz AND foo=bar. This is the behaviour that I want and have used for a long time.
Since 1.91.2, I think extra_label works in an OR way, so alerts fire if abc=xyz OR foo=bar is present.

Is there a way to go back to the AND-behaviour?

Thanks and best regards

Robin

### To Reproduce

1. Set up vmalert (pre 1.91.1) with an alertgroup and a firing test alert.
2. configure multiple extra_label filters. One that the firing alert will match, and one that it wont.
3. Check if alert is still firing (it should not)
4. Update vmalert to 1.91.2 and check again. The alert is firing all of a sudden.

### Version

last working version: 1.91.0
not working from: 1.91.2

### Logs

-

**Comment by hagen1778** [comment]
@robasp thanks for report!
The issue should be addressed at https://github.com/VictoriaMetrics/VictoriaMetrics/pull/4955. Meanwhile you can try [building vmalert from sources](https://docs.victoriametrics.com/vmalert.html#how-to-build-from-sources).
It is also allowed to run vmalert of different version comparing to other cluster components.

**Comment by valyala** [comment]
The old behavior for the extra labels in vmalert config should be restored in the release [v1.93.4](https://docs.victoriametrics.com/CHANGELOG.html). Closing the issue as fixed then.

**Comment by valyala** [comment]
The issue has been fixed also in [v1.87.9 LTS release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.87.9).
